frontend/src/metabase/admin/people/containers/UserSuccessModal.jsx

```diff
@@ -31,35 +35,32 @@ class UserSuccessModal extends React.Component {
         {temporaryPassword ? (
           <PasswordSuccess user={user} temporaryPassword={temporaryPassword} />
         ) : (
-          <EmailSuccess user={user} />
+          <EmailSuccess isEeSsoConfigured={isEeSsoConfigured} user={user} />
         )}
       </ModalContent>
     );
   }
 }
 
-export default _.compose(
-  User.load({
-    id: (state, props) => props.params.userId,
-  }),
-  connect(
-    (state, props) => ({
-      temporaryPassword: getUserTemporaryPassword(state, {
-        userId: props.params.userId,
-      }),
-    }),
-    {
-      onClose: () => push("/admin/people"),
-      clearTemporaryPassword,
-    },
-  ),
-)(UserSuccessModal);
-
-const EmailSuccess = ({ user }) => (
-  <div>{jt`We’ve sent an invite to ${(
-    <strong>{user.email}</strong>
-  )} with instructions to set their password.`}</div>
-);
+const EmailSuccess = ({ user, isEeSsoConfigured }) => {
+  if (isEeSsoConfigured) {
+    return (
+      <div>{jt`We’ve sent an invite to ${(
```

Comment:
Update copy [according to the product doc](https://www.notion.so/metabase/Reconsider-how-we-email-new-users-when-SSO-is-active-f3f51ab268ca4c768721fee7d0e0ef03#c253d985e435475d837efa92a48289be).